enterprise/app/models/enterprise/agent_capacity_policy.rb

```diff
@@ -0,0 +1,145 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: enterprise_agent_capacity_policies
+#
+#  id              :bigint           not null, primary key
+#  account_id      :bigint           not null
+#  name            :string(255)      not null
+#  description     :text
+#  exclusion_rules :jsonb            default: {}
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#
+# Indexes
+#
+#  index_enterprise_agent_capacity_policies_on_account_id          (account_id)
+#  unique_capacity_policy_name_per_account                         (account_id,name) UNIQUE
+#
+# Foreign Keys
+#
+#  fk_rails_...  (account_id => accounts.id)
+#
+
+class Enterprise::AgentCapacityPolicy < ApplicationRecord
```

Comment:
Move this to enterprise/app/models. (Currently it is in enterprise/app/models/enterprise)